docs/source/algorithms.rst

```diff
@@ -50,3 +50,85 @@ Using the same example file as before, we can execute a GaussFilter on our test
     gf.filterExperiment(exp)
     # MzMLFile().store("test.filtered.mzML", exp)
 
+
+
+Spectra Merge Algorithm
+*************************
+
+The pyOpenMS spectra merge algorithms allows the merge of several spectra by either increasing S/N ratio (for MS1 and above) or merging scans which stem from similar precursors (for MS2 and above). In any case, the number of scans will be reduced. In addition, pyOpenMS provides the method to average spectra. 
```

Comment:
Maybe "In addition, pyOpenMS provides the method to average spectra" is a bit unclear. I don't know the details here.